### PR TITLE
fix: infuence-vault show all fees on deposit

### PR DESCRIPTION
**Pull request by mrbasado**

### CODE REVIEW

**Comment by mrbasado** [comment]
Screen Shot 2022-08-26 at 9 59 39 AM
Screen Shot 2022-08-26 at 9 59 49 AM
Screen Shot 2022-08-26 at 10 00 51 AM

File: src/components-v2/landing/VaultListItem.tsx [hunk 1]
```diff
@@ -72,6 +72,7 @@ const VaultListItem = observer(({ vault }: VaultListItemProps): JSX.Element | nu
   const { vaults } = useContext(StoreContext);
   const { depositBalanceDisplay } = useVaultInformation(vault);
   const isTablet = useMediaQuery(useTheme().breakpoints.only('md'));
+  console.log(vault.name, vault);
```
Comment (mrbasado) [inline src/components-v2/landing/VaultListItem.tsx:75]: whoops will remove

File: src/components-v2/common/dialogs/VaultDeposit.tsx [hunk 2]
```diff
@@ -296,7 +300,11 @@ export const VaultDeposit = observer(({ open = false, vault, depositAdvisory, on
             </PercentagesContainer>
           </Grid>
         </Box>
-        <VaultFees vault={vault} className={classes.fees} onHelpClick={() => setShowFees(true)} />
+        {isInfluence && influenceVaultConfig !== undefined ? (
```
Comment (axejintao) [inline src/components-v2/common/dialogs/VaultDeposit.tsx:303]: should this throw an error? 
might be a good follow up - don't want to have partially defined influence data

**Review by axejintao — APPROVED** [review]

File: src/components-v2/common/dialogs/VaultDeposit.tsx (hunk 2 above)
Comment (mrbasado) [reply src/components-v2/common/dialogs/VaultDeposit.tsx:303]: probably? would get a report either way, want me to add or follow up and merge this in?

**Pull request merged**